### DISCUSSION TITLE
[杂谈]邪教用卷土重来了？！

### DISCUSSION THREAD

**Discussion by 550w-1903** [share121/inter-knot#1143] [其他]
![y8gi](https://github.com/user-attachments/assets/2742f263-09d9-4b1a-bc3e-ca456e1767cd)

最近听人说十三分街有人组织进行邪教活动，治安局把附近街的警力全调来了...保真吗？？^－_－^

我一个朋友刚租的门面房，打算卖“保健品”的，不会要创业未半，而中道崩殂了吧？^_^¦¦¦

**Comment by Dave-12138** [comment]
你看见有uploading就说明图还没上传完，要等上传完再点提交

**Comment by 550w-1903** [comment]
谢谢一楼

**Comment by GanlinSama** [comment]
真的假的？

**Comment by gaojiaobei** [comment]
这个委托我接了！
